docs/proposals/PROPOSAL_A.md

```diff
@@ -0,0 +1,293 @@
+# Proposal A
+
+This proposal defines a new artifact manifest and corresponding referrers extension API for registries to support reference types.
+
+The modifications include:
+
+- [New artifact manifest](#description)
+- [Descriptor (OPTIONAL) additional property](#descriptor-properties)
+- [Distribution - Referrers API](#registry-http-api)
+
```

Comment:
again, just a short line here which will fit neatly into the table here: https://github.com/opencontainers/wg-reference-types#proposals

here is the template to refer to: https://github.com/opencontainers/wg-reference-types/blob/main/docs/TEMPLATE.md

Reply:
I'd suggest we leave the content in as there hasn't been a question that wasn't answered here. From what we're seeing, folks ask questions for what's in the doc, and aren't clicking all the links. 
See [here](https://github.com/opencontainers/wg-reference-types/pull/29#discussion_r832543696) for an example where I removed content from the ORAS Artifact spec, that only causes folks to ask questions, respond, them re-read.

I'll add the readme.md change.

Reply:
I'm simply asking that this top section be no more than a few words. Please see the other proposals.

I'll assume you're responding to my comment on simplification. This is your call entirely.

IMO, if proposals are generating questions, this actually results in further collaboration and innovation within the WG vs. us blindly picking one proposal and running with it.

Reply:
> I'm simply asking that this top section be no more than a few words. Please see the other proposals.

Does [this](https://github.com/opencontainers/wg-reference-types/pull/29/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R29) work?